### PR TITLE
feat(InteractionPresets): Add first person interaction preset

### PR DESCRIPTION
**Pull request by psavery**
The first person interaction preset uses the keyboard for movement, and the mouse for "free look"
style camera interactions. This PR also provides a way to access the keyboard manipulator so that the movement speed can be set externally.

### CODE REVIEW

File: Sources/Interaction/Style/InteractorStyleManipulator/index.js [hunk 1]
```diff
@@ -723,6 +723,7 @@ export function extend(publicAPI, model, initialValues = {}) {
 
   // Create get-set macros
   macro.setGet(publicAPI, model, ['rotationFactor']);
+  macro.get(publicAPI, model, ['keyboardManipulators']);
```
Comment (jourdain) [inline Sources/Interaction/Style/InteractorStyleManipulator/index.js]: add the full set `['mouseManipulators', 'keyboardManipulators', 'vrManipulators', 'gestureManipulators']`

Comment (psavery) [reply Sources/Interaction/Style/InteractorStyleManipulator/index.js]: @jourdain Done.

**Comment by jourdain** [comment]
LGTM

**Comment by floryst** [comment]
FYI The failing checks can be ignored, since it looks like some transient issue with building v8.

**Comment by psavery** [comment]
Okay. I think this is ready for merging.

**Pull request merged**